codalab/lib/metadata_util.py

```diff
@@ -60,7 +60,7 @@ def add_edit_argument(parser):
     )
 
 
-def request_missing_metadata(bundle_subclass, args, initial_metadata=None):
+def request_missing_metadata(bundle_subclass, args, initial_metadata=None, bypass_editor= False):
```

Comment:
It might also help to add a brief comment about how `bypass_editor` should affect behavior here.